torchaudio/csrc/rnnt/gpu/compute_alphas.cu

```diff
@@ -0,0 +1,73 @@
+#include <THC/THC.h>
```

Comment:
Do you really need to include THC?

Reply:
I can replace THC with something like `<c10/cuda/CUDAStream.h>` to get the same result -- is there a standard for supporting the `getCurrentCUDAStream` operator?

Reply:
Including <c10/cuda/CUDAStream.h> is fine. THC is legacy and on its way out.